### PR TITLE
[GPII-4331] Create web-developers group and grant log viewer perms on staging

### PR DESCRIPTION
**Pull request by amatas**

### CODE REVIEW

File: common/modules/gcp-project/main.tf [hunk 1]
```diff
@@ -503,6 +515,9 @@ locals {
   # Project owners will be empty list if var.project_owner is empty string ""
   project_owners = "${compact(list(var.project_owner))}"
 
+  # Groups that can see the logs in staging.
```
Comment (natarajaya) [inline common/modules/gcp-project/main.tf]: This may be not only about logs, any other perms that devs may want on stg.

**Comment by natarajaya** [comment]
LGTM

File: common/modules/gcp-project/main.tf (hunk 1 above)
Comment (amatas) [reply common/modules/gcp-project/main.tf]: https://issues.gpii.net/browse/GPII-4331 according with the description of the issue it is the only permission needed. I would close this as it is, and reopen in the case that anything else is needed.

**Comment by stepanstipl** [comment]
LGTM

**Pull request merged**